src/messaging/ExchangeContext.cpp

```diff
@@ -469,5 +466,16 @@ CHIP_ERROR ExchangeContext::HandleMessage(const PacketHeader & packetHeader, con
     return err;
 }
 
+void ExchangeContext::MaybeAutoClose()
+{
+    // Don't close ourselves if we're already closed.
```

Comment:
The comment says 'already closed' but the code reads 'already closed or expected response/send'.

I think we can maybe drop the comment since the code is already clear enough.

Reply:
OK, will drop comment.